dora/core/common/src/main/java/alluxio/membership/EtcdMembershipManager.java

```diff
@@ -194,7 +195,7 @@ public String showAllMembers() {
 
   @Override
   public void stopHeartBeat(WorkerInfo worker) throws IOException {
-    WorkerServiceEntity entity = new WorkerServiceEntity(worker.getAddress());
+    WorkerServiceEntity entity = new WorkerServiceEntity(worker.getIdentity(), worker.getAddress());
```

Comment:
https://github.com/Alluxio/alluxio/pull/18210/files#r1336587495 with constructor, this change can be avoided.